.nvmrc

```diff
@@ -0,0 +1 @@
+v10.0.0
```

Comment:
Can we change that to "10"? nvm should be able to resolve that to the latest version of 10.
```suggestion
10
```